Src/Support/Google.Apis.Auth/OAuth2/GoogleAuthConsts.cs

```diff
@@ -71,5 +74,29 @@ public static class GoogleAuthConsts
 
         /// <summary>Installed application localhost redirect URI.</summary>
         public const string LocalhostRedirectUri = "http://localhost";
+
+        /// <summary>
+        /// The effective Compute Engine authorization token server URL.
+        /// This takes account of the GCE_METADATA_HOST environment variable.
+        /// </summary>
+        internal static string EffectiveComputeTokenUrl => GetEffectiveMetadataUrl(ComputeTokenUrlSuffix, ComputeTokenUrl);
```

Comment:
nit: this might be a little more readable (and same with the Oidc one)?
```suggestion
        internal static string EffectiveComputeTokenUrl => EffectiveMetadataUrl + ComputeTokenUrlSuffix;
```

Reply:
The intention is to use the constant without performing any string allocation when the environment variable isn't set.